models/staging/dbt_utils_day.sql

```diff
@@ -1,10 +1,10 @@
 {% set my_query %}
-    select cast({{dbt_utils.current_timestamp()}} as date)
```

Comment:
@eduardo-nercolini  wont this break compatibilty in diferent sql engines?

Reply:
It's a good question. I would say that the `current_timestamp()` function is a standard SQL function and is widely supported across most SQL database engines, and it's unlikely to "break" in these engines in the sense of causing errors or unexpected behavior due to its widespread adoption and conformity to SQL standards. However, there might be some cases that I'm not aware of where its behavior could cause some errors. I'm open to suggestions for addressing this issue.